FEATURES.md

```diff
@@ -0,0 +1,87 @@
+Some things might be good ideas but they might not fit either with the project as a whole in the specific repo in which the issue was posted. In the latter case, the maintainers will help you determine the right home and transfer the issue.
+- __Is the proposed feature sufficiently distinct from existing features?__
+With a project like OpenSearch it’s totally normal to not know every capability, so this criteria is designed to suss out if the feature provides unique capabilities when compared to existing functionality.
+- __Does the proposed feature overlap or conflict with ongoing work?__
+The best use of everyone’s time is to ensure that there are no competing implementations of features, so this criteria establishes that there is not more than one effort going on for any given area.
+- __Is the feature proposal sufficiently atomic as a feature?__
+Sometimes big ideas should be broken up into smaller ideas. Complete atomicity is probably unobtainable given interdependencies between different facets of the product however the goal here is to make sure that the proposed features are accomplishable in a reasonable amount of time or if it would make more sense to break up the feature into smaller chunks.
+
+The maintainers of the repo decide when and if the proposal meets the criteria. If the feature proposal doesn’t meet the criteria, then the issue will be closed with an explanation and tagged clearly. If the feature does meet the criteria and someone is volunteering to work on the feature, it moves to the next stage. If the feature still needs someone to write the feature, it is tagged as “help wanted” until someone volunteers.
+
+In this process your issue might be closed for a variety of reasons and it doesn't necessarily mean that it’s a bad idea or that the feature isn’t wanted. It might mean that the way it is presented, supported, or the context of the other development in the project doesn’t align with the feature proposal process. If this is the case, the maintainer can help with next steps to move forward.
+
+__Creating META issue(s):__
+
+Once the feature proposal is approved, the next step is to create a parent META issue detailing the plan to implement this feature. The parent META issue should contain an overview section along with associated tasks, link to design document, original proposal and acceptance criteria. Sometimes it's very difficult to represent all the above details in a parent META issue especially when dealing with features that span multiple release versions (i.e. not atomic). In those cases, it's recommended to create a child META issue and link it to the parent META issue for tracking. The child META issue will follow the same format as the parent META issue but will track only the tasks targeted for specific OpenSearch version. The Maintainer of the repo will add the META issue associated with a specific release version to the release [roadmap](https://github.com/orgs/opensearch-project/projects/1).
```

Comment:
Is it really ever "approved"? Maybe "Once the maintainers generally agree that the feature is valuable"?

Reply:
Lowercase META, that's a company ticker. Use "meta-issue".